eyeshade/lib/transaction.js

```diff
@@ -1,27 +1,98 @@
 const BigNumber = require('bignumber.js')
 const getPublisherProps = require('bat-publisher').getPublisherProps
 const uuidv5 = require('uuid/v5')
+const uuid = require('uuid')
```

Comment:
can you change this in light of the notice on https://www.npmjs.com/package/uuid
> [Deprecation warning: The use of require('uuid') is deprecated and will not be supported after version 3.x of this module. Instead, use require('uuid/[v1|v3|v4|v5]') as shown in the examples below.]

Reply:
mmm. makes sense. can do. i'll have to make sure that that is not happening in other prs too